src/python/grpcio/grpc/_cython/_cygrpc/aio/channel.pyx.pxi

```diff
@@ -28,11 +38,12 @@ cdef class AioChannel:
 
     def call(self,
              bytes method,
-             object deadline):
+             object deadline,
+             CallCredentials credentials):
```

Comment:
nit: we could call it `call_credentials`, so we don't mix it with ChannelCredentials.